sql/sp_instr.h

```diff
@@ -242,9 +242,9 @@ class sp_lex_instr : public sp_instr
   virtual SQL_I_List<Item_trigger_field>* get_instr_trig_field_list()
   { return &m_trig_field_list; }
 
-  int get_command() const {
-      return m_lex ? m_lex->sql_command : -1;
-  }
+  int get_command() const { return m_lex ? m_lex->sql_command : -1; }
```

Comment:
This must be inside `#ifdef WITH_WSREP`

Reply:
Why? I would rather say: "please revert formating change" :)
This code is from PS

Reply:
Sorry for mistakenly commenting on `get_command()`. I intended to make the below `get_lex()` function be enclosed under `#ifdef WITH_WSREP` as this change is a PXC specific change.

Reply:
Done (get_lex I suppose, get_command is just reformatted).

Reply:
Can you please revert the formatting changes done for the `get_command()` function? This will cause merge conflict when it is changed in PS in future.

```suggestion
  int get_command() const
  { 
    return m_lex ? m_lex->sql_command : -1; 
  }
```

Reply:
Done